dev-docs/source/ObtainingABenchmarkForMantidFitting.rst

```diff
@@ -0,0 +1,104 @@
+.. _ObtainingABenchmarkForMantidFitting:
+
+Obtaining a Benchmark for Mantid Fitting
+========================================
+
+.. contents::
+  :local:
+
+There are several scenarios in which obtaining a benchmark for the accuracy and runtime of Mantid minimizers might be useful:
+
+- From the perspective of a **Scientist**, who wants to know the best Mantid minimizer to use for fitting their model to their data.
+- From the perspective of a **Software Developer**, who wants to optimize their code to improve its accuracy and runtime.
+
+This page will explain how to set up a python virtual environment that can be used to benchmark the accuracy and runtime of different fitting minimizers in Mantid for different fitting problems. To do this, we will use a package called `FitBenchmarking <https://fitbenchmarking.readthedocs.io/en/stable/>`_ which is a cross-platform open source tool for comparing different minimizers and fitting frameworks.
+
+Initial Setup
+#############
+
+Some initial setup is required on your system before we can start setting up an environment for benchmarking Mantid minimizers. Before you start this process, make sure you have a recent version of python installed and access to a terminal or command prompt.
+
+1. Download and install the desired version of Mantid from the `downloads <https://download.mantidproject.org/>`_ page (if not already installed). Note its installation path, hereby denoted as ``[INSTALL_PATH]``.
+2. Open a terminal or command prompt and cd to the desired location.
+3. Pip install the virtual environment package as described in the 'Installing virtualenv' section found `here <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv>`_.
+
+Creating your Benchmarking Environment
+######################################
+
+1. From the same terminal or command prompt, run the following command:
+
+If using Windows:
+
+.. code-block:: sh
+
+  py -m virtualenv -p=[INSTALL_PATH]\bin\python.exe benchmark-env
+
+This will create a virtual environment based on the python executable provided with your Mantid installation. For Windows, the location of this executable is ``C:\MantidInstall\bin\python.exe``.
+
+If using Ubuntu:
+
+.. code-block:: sh
+
+  virtualenv --python=python3 benchmark-env
+
+In this example, ``benchmark-env`` is the name given to your virtual environment.
+
+2. Activate your environment. This is explained in the 'Activating a virtual environment' section found `here <https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#activating-a-virtual-environment>`_.
+
+3. It might be necessary to upgrade and install a few packages before installing FitBenchmarking if working from a Linux terminal:
+
+.. code-block:: sh
```

Comment:
In the html version of the documentation this code block is missing.

Reply:
Thanks, that should be fixed now